### DISCUSSION TITLE
Breadcrumbs Guides & Content

### DISCUSSION THREAD

**Discussion by SkepticMystic** [SkepticMystic/breadcrumbs#129] [Show and tell]
This discussion is for any guides or content related to Breadcrumbs outside of the (outdated) readme and (better) [wiki](https://github.com/SkepticMystic/breadcrumbs/wiki).

Please do let me know if you make any content related to Breadcrumbs and would like to post it here 😄

**Comment by SkepticMystic** [comment]
@vizance has written two Medium articles about Breadcrumbs! 😄 
You can find them here (note that you may need to translate the page, Google Translate can do so):
- [1](https://medium.com/pm%E7%9A%84%E7%94%9F%E7%94%A2%E5%8A%9B%E5%B7%A5%E5%85%B7%E7%AE%B1/obsidian-%E4%BD%BF%E7%94%A8%E6%95%99%E5%AD%B8-%E6%8F%92%E4%BB%B6%E7%AF%87-03-%E5%A6%82%E4%BD%95%E8%AE%93-obsidian-%E8%87%AA%E5%8B%95%E6%8E%A8%E8%96%A6%E9%97%9C%E8%81%AF%E7%AD%86%E8%A8%98-4d9acb52d059)
- [2](https://medium.com/pm%E7%9A%84%E7%94%9F%E7%94%A2%E5%8A%9B%E5%B7%A5%E5%85%B7%E7%AE%B1/obsidian-%E4%BD%BF%E7%94%A8%E6%95%99%E5%AD%B8-%E6%87%89%E7%94%A8%E7%AF%87-04-%E5%A6%82%E4%BD%95%E8%AE%93-obsidian-%E8%87%AA%E5%8B%95%E6%8E%A8%E8%96%A6%E9%97%9C%E8%81%AF%E7%AD%86%E8%A8%98-c36a80707f85)

**Reply by ooker777** [reply]
Medium requires you to pay for reading. Plus that it's occasionally blocked in my country

**Comment by SkepticMystic** [comment]
I have gave a [Community Showcase](https://forum.obsidian.md/t/meta-community-talks/16686) talk about Breadcrumbs which you can watch [here](https://www.youtube.com/watch?v=DXXB7fHcArg).

**Comment by SkepticMystic** [comment]
@lkadre's guide on Getting the most out of Breadcrumbs is great!
https://github.com/SkepticMystic/breadcrumbs/discussions/175
